crates/rspack_core/src/artifacts/mod.rs

```diff
@@ -14,6 +15,7 @@ mod side_effects_do_optimize_artifact;
 pub use cgm_hash_artifact::*;
 pub use cgm_runtime_requirement_artifact::*;
 pub use chunk_hashes_artifact::*;
+pub use chunk_render_cache_artifact::*;
```

Comment:
do not export all mods, export what you use only.

Reply:
yes!